docs/GettingStarted.md

```diff
@@ -81,21 +81,35 @@ jest --init
 
 ### Using Babel
 
-To use [Babel](http://babeljs.io/), install the `babel-jest` and `@babel/core` packages via `yarn`:
+To use [Babel](http://babeljs.io/), install required development dependencies via `yarn`:
 
 ```bash
-yarn add --dev babel-jest @babel/core
+yarn add --dev @babel/core @babel/preset-env @babel/preset-react
 ```
 
-Don't forget to add a [`babel.config.js`](https://babeljs.io/docs/en/config-files) file in your project's root folder. For example, if you are using ES2018 and [React](https://reactjs.org) with the [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env) and [`@babel/preset-react`](https://babeljs.io/docs/en/babel-preset-react) presets:
+Install `@babel/polyfill`:
```

Comment:
I don't think we should mention `@babel/polyfill` at all - it's not relevant to us.

Reply:
The idea was that using `@babel/polyfill` would be appropriate for a React app, and this setup already includes setting up `@babel/preset-react`.

If we want this focused on Node maybe both should be removed.